FSharpLu.Tests/JsonTests.fs

```diff
@@ -35,22 +36,22 @@ module SomeAmbiguity =
 let inline defaultSerialize< ^T> (x: ^T) = Compact.serialize< ^T> x
 
 let inline reciprocal< ^T when ^T:equality> (serialize: ^T->string) (deserialize: string-> ^T) (x: ^T) =
-    // theoretically one round trip is sufficient; we perform 
+    // theoretically one round trip is sufficient; we perform
     // two round trips here to test for possible side-effects
     x |> serialize |> deserialize |> serialize |> deserialize = x
 
-let inline canBeSerialized< ^T when ^T:equality> (serialize: ^T->string) (deserialize: string-> ^T) (x: ^T) = 
+let inline canBeSerialized< ^T when ^T:equality> (serialize: ^T->string) (deserialize: string-> ^T) (x: ^T) =
```

Comment:
Just a note (not really actionable) - since the constraint being applied to `T` isn't a static member, but rather something that is "native" to .NET (it implies `System.IComparable`).

In fact, the SRTP syntax doesn't even need to be written even if you'd think that it would, despite being inferred correctly:

```fsharp
let inline test<'T when 'T:(static member Uhh:'T)> (x: 'T) = ()
// val inline test : x: ^T -> unit when  ^T : (static member get_Uhh : ->  ^T)

let inline test'<'T when 'T: equality> (x: 'T) = ()
// val inline test' : x:'T -> unit when 'T : equality
```

I believe this is actually a bug in F#, but it does mean that the caret syntax isn't necessary.